src/main/java/frc/robot/Constants.java

```diff
@@ -95,42 +96,42 @@ public static class DriveConstants {
   }
 
   public static final class SwerveModuleConstants {
-      public static final double kMaxSpeedMetersPerSecond = DriveConstants.kMaxSpeedMetersPerSecond * 2.0;
-
-      public static final double kAbsoluteEncoderCPR = 4096.0;
-      public static final double kWheelDiameterMeters = 0.09525;
-
-      public static final double kDriveGearRatio = 6.75;
-      public static final double kTurningGearRatio = 150.0 / 7.0;
-
-      // The native position units are motor rotations, but we want meters.
-      public static final double kDrivePositionConversionFactor =
-        (SwerveModuleConstants.kWheelDiameterMeters * Math.PI)
-        / SwerveModuleConstants.kDriveGearRatio;
-      // The native velocity units are motor rotations [aka revolutions] per minute (RPM),
-      // but we want meters per second.
-      public static final double kDriveVelocityConversionFactor =
-        kDrivePositionConversionFactor
-        / 60.0 /* s */;
-
-      // The native position units are motor rotations, but we want radians.
-      public static final double kTurningPositionConversionFactor =
-        Units.rotationsToRadians(1.0 / SwerveModuleConstants.kTurningGearRatio);
-      // The native velocity units are motor rotations [aka revolutions] per minute (RPM), but we want radians per second.
-      public static final double kTurningVelocityConversionFactor =
-        kTurningPositionConversionFactor
-        / 60.0 /* s */;
-
-      public static final PID kDrivePID = new PID(0.5, 0.0, 0.0);
-      public static final PID kTurningPID = new PID(0.5, 0.0, 0.0);
-
-      public static final int kDriveMotorCurrentLimit = 40;
-      public static final int kTurningMotorCurrentLimit = 20;
-
-      public static final double kDriveMotorRampRate = 0.25;
-      public static final double kTurningMotorRampRate = 0.25;
-
-      public static final long kValueCacheTtlMicroseconds = 15;
+    public static final double kMaxSpeedMetersPerSecond = DriveConstants.kMaxSpeedMetersPerSecond * 2.0;
+
+    //TODO: Find actual wheel diameter
+    public static final double kWheelDiameterMeters = 0.09924;
```

Comment:
Why did this number change if we still don't know the true value?

Reply:
It is the measured ideal diameter, during our conversation you suggested a better way to find diameter, so this is just an ideal diameter for right now.